liesel/model/model.py

```diff
@@ -799,6 +828,17 @@ def transform(
                 "and no bijector was given"
             )
 
+        using_bijector_instance = not use_default_bijector and not isinstance(
+            bijector, type
+        )
+
+        if isinstance(bijector, type) and not args and not kwargs:
```

Comment:
might use

`if isinstance(bijector, type) and not (args or kwargs):`

for better readability.